catroid/src/androidTest/java/org/catrobat/catroid/test/content/actions/GoNStepsBackActionTest.java

```diff
@@ -67,12 +68,15 @@ public void setUp() throws Exception {
 
 		background = new SingleSprite("background");
 		parentGroup.addActor(background.look);
-		sprite = new SingleSprite("testSprite");
+		sprite = new SingleSprite("penActor");
```

Comment:
please also rename the sprites accordingly, it would make the test way more readable further down...